docs/reference/dep-007.md

```diff
@@ -0,0 +1,61 @@
++++
+dep = 7
+title = "Draft Init Drivers"
+authors = [ "Matt Fisher <matt.fisher@microsoft.com>" ]
+created = 2017-02-28
++++
+
+# Introduction
+
+Draft Init Drivers are a special type of [plugin][]. They initialize Draftd on a given cloud provider utilizing their best-of-breed technology, released on a separate schedule than Draft's core.
+
+# Abstract
+
+Draft Init Drivers are plugins that integrate seamlessly with Draft. They provide a way to install the Draft server on a given cloud provider without requiring every cloud provider to contribute to the core tool.
+
+New drivers are installed using the `draft plugin install` command. For example,
+
+```bash
+$ draft plugin install https://bacongobbler/draft-init-minikube
+```
+
+Will install the `minikube` driver.
+
+To run `draft init` with a driver, provide the name of the driver with the `--driver` flag. With the previous example, this can be called by running
+
+```bash
+$ draft init --driver=minikube
+```
+
+By not providing the `--driver` flag, `draft init` will use the existing behaviour, so these drivers are purely opt-in. Should drivers prove to be the best way forward for `draft init`, this may eventually become the default at a later time.
+
+## Writing a Driver
+
+Writing a new driver for Draft is the same as writing a new plugin, with a few caveats:
+
+1. The name of the plugin must be `draft-init-drivername`
+1. The plugin accepts a single argument: the local path to the chart containing `draftd`
```

Comment:
hey I'm a little confused on this one. Is the chart shipped with the plugin?

Reply:
I can definitely elaborate on this a little more in the docs, but basically Draft will essentially perform a `helm package`, merging values provided through the CLI option flags like `--ingress-enabled=true` into the chart's values.yaml. After packaging the chart, draft invokes the plugin and provides the local path to the packaged chart as the plugin's first argument.

Does that help clarify a little better?

Reply:
And yes, it is the responsibility of the plugin to install the chart onto the cluster.

Reply:
Added more info in the docs to hopefully elaborate on this a little more. Let me know if there's more knowledge gaps I need to fill!